core/quantity/src/Formatter/Format.ts

```diff
@@ -316,6 +350,37 @@ export class Format extends BaseFormat {
       if (undefined === this.units || this.units.length === 0)
         throw new QuantityError(QuantityStatus.InvalidJson, `The Format ${this.name} has a Composite with no valid 'units'`);
     }
+
+    if(this.type === FormatType.Azimuth || this.type === FormatType.Bearing) {
+      // these units cannot be loaded from loadFormatProperties() because they require an async call, and the method signature is already public
+
+      if (undefined !== jsonObj.azimuthBaseUnit) {
+        if (typeof (jsonObj.azimuthBaseUnit) !== "string")
```

Comment:
type validation is already ddone in loadFormatProperties() above - do we need to do this again?

Reply:
My bad, I didn't see this is for azimuthBaseUnit, not azimuthBase @@